bigbluebutton-html5/imports/ui/components/webcam/component.jsx

```diff
@@ -232,7 +233,7 @@ const WebcamComponent = ({
             }}
             style={{
               position: 'absolute',
-              zIndex: isCameraSidebar ? 0 : cameraDock.zIndex,
+              zIndex: cameraDock.zIndex,
```

Comment:
```suggestion
              zIndex: isCameraSidebar && !isDragging ? 0 : cameraDock.zIndex,
```
zIndex 0 is needed because of https://github.com/bigbluebutton/bigbluebutton/pull/17759